src/python/ensembl/io/genomio/seq_region/rename.py

```diff
@@ -0,0 +1,212 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Rename seq_region BRC names in a given core database."""
+
+from dataclasses import dataclass
+import logging
+from enum import Enum, auto
+from pathlib import Path
+from typing import Any, Dict, List
+
+from sqlalchemy import select, or_
+from sqlalchemy.orm import Session
+
+from ensembl.core.models import SeqRegion, SeqRegionSynonym, SeqRegionAttrib
+from ensembl.database import DBConnection
+from ensembl.utils.argparse import ArgumentParser
+from ensembl.utils.logging import init_logging_with_args
+
+
+class Operation(Enum):
+    """Controlled list of operations for the seq_region replacement."""
+
+    do_nothing = auto()
+    update = auto()
+    insert = auto()
+    not_found = auto()
```

Comment:
Due to their nature of being constants, `Enum` attributes are strongly recommended to be written in upper case, e.g. `DO_NOTHING`.